components/help-and-support/ContactSection.js

```diff
@@ -66,10 +67,21 @@ const ContactForm = () => {
       topic: '',
       message: '',
       link: '',
+      relatedCollectives: [],
     },
     validate,
     onSubmit: values => {
       setIsSubmitting(true);
+      if (values.relatedCollectives.length === 0 && LoggedInUser) {
+        setFieldValue(
+          'relatedCollectives',
+          LoggedInUser.memberOf.map(member => {
+            if (member.role === 'ADMIN') {
+              return `https://opencollective.com/${member.collective.slug}`;
```

Comment:
Let's not hardcode the domain

```suggestion
              return getCollectivePageCanonicalURL(member.collective);
```